src/Infrastructure/Identity/RoleService.cs

```diff
@@ -95,12 +96,16 @@ public async Task<List<RoleDto>> GetListAsync()
         return roleDtos;
     }
 
-    public async Task<List<PermissionDto>> GetPermissionsAsync(string id, CancellationToken cancellationToken)
+    /// <summary>
+    /// Get Permissions By Role Async.
+    /// </summary>
+    /// <param name="roleId"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    public async Task<List<PermissionDto>> GetPermissionsByRoleAsync(string roleId, CancellationToken cancellationToken)
```

Comment:
I don't think the name needed changing... the fact that this is a method of the "roleservice" is enough to denote it's about roles, no?